specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2020-10-20/examples/WorkbookManagedAdd.json

```diff
@@ -0,0 +1,110 @@
+{
+  "parameters": {
+    "api-version": "2020-10-20",
+    "resourceGroupName": "my-resource-group",
+    "subscriptionId": "6b643656-33eb-422f-aee8-3ac145d124af",
+    "resourceName": "deadb33f-5e0d-4064-8ebb-1a4ed0313eb2",
+    "sourceId": "/subscriptions/00000000-0000-0000-0000-00000000/resourcegroups/MyGroup",
+    "workbookProperties": {
+      "id": "/subscriptions/00000000-0000-0000-0000-00000000/resourcegroups/MyGroup/providers/microsoft.insights/workbooks/e378d137-1243-4256-b5c4-ad2a937cae79",
+      "name": "e378d137-1243-4256-b5c4-ad2a937cae79",
+      "location": "westus",
+      "tags": {
+        "hidden-title": "tttt"
+      },
+      "kind": "user",
+      "etag": "\"4a00f78d-0000-0700-0000-5f8f616c1000\"",
+      "identity": {
+        "type": "UserAssigned",
+        "userAssignedIdentities": {
+          "/subscriptions/00000000-0000-0000-0000-00000000/resourceGroups/eu2cgroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id1": {},
+          "/subscriptions/00000000-0000-0000-0000-00000000/resourceGroups/eu2cgroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/id2": {}
```

Comment:
fyi.. we only support a single identity at this time, if user adds multiple then it would return an error